estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class EstatePropertyTag(models.Model):
+    _name = 'estate.property.tag'
+    _description = "Real Estate Property Tag"
+    _order = 'name asc'
```

Comment:
```suggestion
    _order = 'name'
```
Do we need to explicitly define `asc`? :thinking: